### PR TITLE
Bazel Build config iOS Tests

### LINKED ISSUE #1301: Bazel config for iOS tests
These tests need to be added to the Bazel build config.

This is the `test` scheme in the legacy Xcode build config.

Sources:

```
platform/ios/platform/ios/test/MLNMockGestureRecognizers.m
platform/ios/platform/ios/test/MLNMockGestureRecognizers.h
platform/ios/platform/darwin/test/MLNTestAssertionHandler.m
platform/ios/platform/darwin/test/MLNDocumentationExampleTests.swift
platform/ios/platform/darwin/test/MLNOfflineRegionTests.m
platform/ios/platform/ios/test/MLNMapViewDelegateIntegrationTests.swift
platform/ios/platform/darwin/test/MLNMapSnapshotterTests.m
platform/ios/platform/darwin/test/MLNStyleTests.mm
platform/ios/platform/darwin/test/MLNGeometryTests.mm
platform/ios/platform/darwin/test/MLNHeatmapStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNHeatmapColorTests.mm
platform/ios/platform/darwin/test/MLNFillStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNCodingTests.mm
platform/ios/platform/darwin/test/MLNDocumentationGuideTests.swift
platform/ios/platform/darwin/test/MLNImageTests.mm
platform/ios/platform/darwin/test/MLNMapViewTests.m
platform/ios/platform/darwin/test/MLNDistanceFormatterTests.m
platform/ios/platform/darwin/test/MLNImageSourceTests.m
platform/ios/platform/darwin/test/MLNStyleLayerTests.m
platform/ios/platform/darwin/test/MLNCompassDirectionFormatterTests.m
platform/ios/platform/darwin/test/MLNNSStringAdditionsTests.m
platform/ios/platform/ios/test/MLNNetworkConfigurationTests.m
platform/ios/platform/darwin/test/MLNTileSetTests.mm
platform/ios/platform/darwin/test/MLNAttributionInfoTests.m
platform/ios/platform/darwin/test/MLNLightTest.mm
platform/ios/platform/darwin/test/MLNSymbolStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNLineStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNCircleStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNShapeSourceTests.mm
platform/ios/platform/darwin/test/MLNClockDirectionFormatterTests.m
platform/ios/platform/darwin/test/MLNPredicateTests.mm
platform/ios/platform/ios/test/MLNMapViewCompassViewTests.mm
platform/ios/platform/ios/test/MLNNSOrthographyAdditionsTests.m
platform/ios/platform/darwin/test/MLNNSDateAdditionsTests.mm
platform/ios/platform/darwin/test/MLNMapCameraTests.m
platform/ios/platform/ios/test/MLNMapViewPitchTests.m
platform/ios/platform/darwin/test/MLNRendererConfigurationTests.m
platform/ios/platform/darwin/test/MLNResourceTests.mm
platform/ios/platform/darwin/test/MLNExpressionTests.mm
platform/ios/platform/darwin/test/MLNBackgroundStyleLayerTests.mm
platform/ios/platform/ios/test/MLNMapViewScaleBarTests.m
platform/ios/platform/ios/test/MLNAnnotationViewIntegrationTests.swift
platform/ios/platform/ios/test/MLNMapViewZoomTests.mm
platform/ios/platform/darwin/test/MLNOfflinePackTests.mm
platform/ios/platform/darwin/test/MLNOfflineStorageTests.mm
platform/ios/platform/darwin/test/MLNComputedShapeSourceTests.m
platform/ios/platform/darwin/test/MLNSourceQueryTests.m
platform/ios/platform/ios/test/MLNMapAccessibilityElementTests.m
platform/ios/platform/ios/test/MLNMapViewDirectionTests.mm
platform/ios/platform/ios/test/MLNMapViewContentInsetTests.m
platform/ios/platform/darwin/test/MLNFillExtrusionStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNCoordinateFormatterTests.m
platform/ios/platform/darwin/test/MLNRasterStyleLayerTests.mm
platform/ios/platform/darwin/test/MLNHillshadeStyleLayerTests.mm
platform/ios/platform/ios/test/MLNMapViewGestureRecognizerTests.mm
platform/ios/platform/ios/test/MLNAnnotationViewTests.m
platform/ios/platform/darwin/test/MLNFeatureTests.mm
platform/ios/platform/darwin/test/MLNVersionNumber.m
platform/ios/platform/darwin/test/MLNSDKTestHelpers.swift
platform/ios/platform/darwin/test/MLNMapProjectionTests.m
platform/ios/platform/ios/test/MLNMapViewLayoutTests.m
```

### PR DESCRIPTION
**Pull request by louwers**
- Add artifact creation & upload for iOS render test `ios-ci.yml` workflow
- As part of the above I upgraded the macOS runner to macOS Ventura
- Since the CMake based iOS tests have some issues on this new runner, I moved the build config of the iOS Tests over to Bazel.
- ~TODO: port iOS UI tests to Bazel~
- TODO: https://github.com/maplibre/maplibre-native/issues/1304

Closes #1301

~Now the tests are failing (again) when I run them through Bazel. Works through Xcode though... Hmm.~

### CODE REVIEW

**Comment by louwers** [comment]
Test fails when run with `bazel test`, passes when ran through Xcode.

Log:

```
2023-07-02 14:34:33,550 Creating a new simulator:
Name: New-iPhone 14 Pro Max-16.4
OS: iOS 16.4
Type: iPhone 14 Pro Max
2023-07-02 14:34:34,277 Created new simulator 9C9E5E8C-69C9-401D-80AA-29E73411471B.
2023-07-02 14:34:34,277 Will consider the test as test type Logic Test to run. Because the app under test is not given.
Test Suite 'All tests' started at 2023-07-02 14:34:35.235
Test Suite 'ios_test.xctest' started at 2023-07-02 14:34:35.236
Test Suite 'MLNAnnotationViewTests' started at 2023-07-02 14:34:35.236
Test Case '-[MLNAnnotationViewTests testAnnotationView]' started.
2023-07-02 14:34:35.273 xctest[66761:1090873] MLNMapView WARNING UIViewController.automaticallyAdjustsScrollViewInsets is deprecated use MLNMapView.automaticallyAdjustContentInset instead.
Invalid connection: com.apple.coresymbolicationd
<unknown>:0: error: -[MLNAnnotationViewTests testAnnotationView] : CALayer position contains NaN: [nan nan]. Layer: <CALayer:0x6000021ea4c0; position = CGPoint (0 0); bounds = CGRect (0 0; 0 0); delegate = <MLNAnnotationView: 0x14a314870; frame = (0 0; 0 0); layer = <CALayer: 0x6000021ea4c0>>; opaque = YES; allowsGroupOpacity = YES; > (CALayerInvalidGeometry)
<unknown>:0: error: -[MLNAnnotationViewTests testAnnotationView] : Failed due to unwaited expectation 'annotation property'.
Test Case '-[MLNAnnotationViewTests testAnnotationView]' failed (0.360 seconds).
Test Case '-[MLNAnnotationViewTests testAnnotationViewInitWithFramePENDING]' started.
Test Case '-[MLNAnnotationViewTests testAnnotationViewInitWithFramePENDING]' passed (0.012 seconds).
Test Case '-[MLNAnnotationViewTests testAnnotationViewInitWithReuseIdentifier]' started.
Test Case '-[MLNAnnotationViewTests testAnnotationViewInitWithReuseIdentifier]' passed (0.007 seconds).
Test Case '-[MLNAnnotationViewTests testCustomAnnotationView]' started.
Test Case '-[MLNAnnotationViewTests testCustomAnnotationView]' passed (0.011 seconds).
Test Case '-[MLNAnnotationViewTests testSelectingADisabledAnnotationViewPENDING]' started.
Test Case '-[MLNAnnotationViewTests testSelectingADisabledAnnotationViewPENDING]' passed (0.012 seconds).
Test Case '-[MLNAnnotationViewTests testSelectingOffscreenAnnotation]' started.
<unknown>:0: error: -[MLNAnnotationViewTests testSelectingOffscreenAnnotation] : CALayer position contains NaN: [nan nan]. Layer: <CALayer:0x6000027cc560; position = CGPoint (0 0); bounds = CGRect (0 0; 0 0); delegate = <MLNAnnotationView: 0x14a42e180; frame = (0 0; 0 0); layer = <CALayer: 0x6000027cc560>>; opaque = YES; allowsGroupOpacity = YES; > (CALayerInvalidGeometry)
Test Case '-[MLNAnnotationViewTests testSelectingOffscreenAnnotation]' failed (0.012 seconds).
Test Case '-[MLNAnnotationViewTests testSelectingOnscreenAnnotationThatHasNotBeenAdded]' started.
<unknown>:0: error: -[MLNAnnotationViewTests testSelectingOnscreenAnnotationThatHasNotBeenAdded] : CALayer position contains NaN: [nan nan]. Layer: <CALayer:0x600002ad4c40; position = CGPoint (0 0); bounds = CGRect (0 0; 0 0); delegate = <MLNAnnotationView: 0x148d0fd70; frame = (0 0; 0 0); layer = <CALayer: 0x600002ad4c40>>; opaque = YES; allowsGroupOpacity = YES; > (CALayerInvalidGeometry)
Test Case '-[MLNAnnotationViewTests testSelectingOnscreenAnnotationThatHasNotBeenAdded]' failed (0.015 seconds).
Test Suite 'MLNAnnotationViewTests' failed at 2023-07-02 14:34:35.667.
	 Executed 7 tests, with 4 failures (4 unexpected) in 0.429 (0.430) seconds
Test Suite 'MLNMapAccessibilityElementTests' started at 2023-07-02 14:34:35.667
Test Case '-[MLNMapAccessibilityElementTests testFeatureLabels]' started.
Test Case '-[MLNMapAccessibilityElementTests testFeatureLabels]' passed (0.017 seconds).
Test Case '-[MLNMapAccessibilityElementTests testPlaceFeatureValues]' started.
Test Case '-[MLNMapAccessibilityElementTests testPlaceFeatureValues]' passed (0.002 seconds).
Test Case '-[MLNMapAccessibilityElementTests testRoadFeatureValues]' started.
Test Case '-[MLNMapAccessibilityElementTests testRoadFeatureValues]' passed (0.001 seconds).
Test Suite 'MLNMapAccessibilityElementTests' passed at 2023-07-02 14:34:35.687.
	 Executed 3 tests, with 0 failures (0 unexpected) in 0.019 (0.020) seconds
Test Suite 'MLNMapViewLayoutTests' started at 2023-07-02 14:34:35.687
Test Case '-[MLNMapViewLayoutTests testAttributionButtonPlacement]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testAttributionButtonPlacement] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:34:45.710 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testAttributionButtonPlacement] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testAttributionButtonPlacement]' failed (10.031 seconds).
Test Case '-[MLNMapViewLayoutTests testAttributionButtonPlacementInvalidPosition]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testAttributionButtonPlacementInvalidPosition] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:34:55.737 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testAttributionButtonPlacementInvalidPosition] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testAttributionButtonPlacementInvalidPosition]' failed (10.027 seconds).
Test Case '-[MLNMapViewLayoutTests testCompassPlacement]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testCompassPlacement] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:35:05.769 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testCompassPlacement] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testCompassPlacement]' failed (10.032 seconds).
Test Case '-[MLNMapViewLayoutTests testCompassPlacementWithTransform]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testCompassPlacementWithTransform] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:35:15.800 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testCompassPlacementWithTransform] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testCompassPlacementWithTransform]' failed (10.031 seconds).
Test Case '-[MLNMapViewLayoutTests testLogoPlacement]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testLogoPlacement] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:35:25.825 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testLogoPlacement] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testLogoPlacement]' failed (10.025 seconds).
Test Case '-[MLNMapViewLayoutTests testLogoPlacementInvalidPosition]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testLogoPlacementInvalidPosition] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:35:35.852 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testLogoPlacementInvalidPosition] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testLogoPlacementInvalidPosition]' failed (10.030 seconds).
Test Case '-[MLNMapViewLayoutTests testOrnamentPlacement]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testOrnamentPlacement] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:35:45.888 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testOrnamentPlacement] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testOrnamentPlacement]' failed (10.038 seconds).
Test Case '-[MLNMapViewLayoutTests testScalebarPlacement]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testScalebarPlacement] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:35:55.921 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testScalebarPlacement] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testScalebarPlacement]' failed (10.033 seconds).
Test Case '-[MLNMapViewLayoutTests testScalebarPlacementInvalidPosition]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testScalebarPlacementInvalidPosition] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:36:05.949 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testScalebarPlacementInvalidPosition] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testScalebarPlacementInvalidPosition]' failed (10.026 seconds).
Test Case '-[MLNMapViewLayoutTests testScalebarSubviewPlacement]' started.
<unknown>:0: error: -[MLNMapViewLayoutTests testScalebarSubviewPlacement] : Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "Map view should finish loading style.".
2023-07-02 14:36:15.976 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewLayoutTests testScalebarSubviewPlacement] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewLayoutTests testScalebarSubviewPlacement]' failed (10.030 seconds).
Test Suite 'MLNMapViewLayoutTests' failed at 2023-07-02 14:36:15.990.
	 Executed 10 tests, with 20 failures (10 unexpected) in 100.302 (100.303) seconds
Test Suite 'MLNMapViewPitchTests' started at 2023-07-02 14:36:15.990
Test Case '-[MLNMapViewPitchTests testHorizontalTiltGesture]' started.
Test Case '-[MLNMapViewPitchTests testHorizontalTiltGesture]' passed (0.011 seconds).
Test Case '-[MLNMapViewPitchTests testNegativePitchClamping]' started.
Test Case '-[MLNMapViewPitchTests testNegativePitchClamping]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testPitchAtVariousZoomLevels]' started.
Test Case '-[MLNMapViewPitchTests testPitchAtVariousZoomLevels]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testPitchEnabled]' started.
Test Case '-[MLNMapViewPitchTests testPitchEnabled]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testPitchInValidRange]' started.
Test Case '-[MLNMapViewPitchTests testPitchInValidRange]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testPositivePitchClamping]' started.
Test Case '-[MLNMapViewPitchTests testPositivePitchClamping]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testTiltGesture]' started.
Test Case '-[MLNMapViewPitchTests testTiltGesture]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testTiltGestureFromInitialTilt]' started.
Test Case '-[MLNMapViewPitchTests testTiltGestureFromInitialTilt]' passed (0.005 seconds).
Test Case '-[MLNMapViewPitchTests testTiltGestureNumberOfFingersRequired]' started.
Test Case '-[MLNMapViewPitchTests testTiltGestureNumberOfFingersRequired]' passed (0.005 seconds).
Test Suite 'MLNMapViewPitchTests' passed at 2023-07-02 14:36:16.043.
	 Executed 9 tests, with 0 failures (0 unexpected) in 0.053 (0.053) seconds
Test Suite 'MLNMapViewScaleBarTests' started at 2023-07-02 14:36:16.043
Test Case '-[MLNMapViewScaleBarTests testDirectlySettingScaleBarViewHiddenProperty]' started.
2023-07-02 14:36:16.047 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewScaleBarTests testDirectlySettingScaleBarViewHiddenProperty] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewScaleBarTests testDirectlySettingScaleBarViewHiddenProperty]' failed (0.008 seconds).
Test Case '-[MLNMapViewScaleBarTests testShowsScale]' started.
/Users/ec2-user/Library/Developer/CoreSimulator/Devices/9C9E5E8C-69C9-401D-80AA-29E73411471B/data/platform/ios/platform/ios/test/MLNMapViewScaleBarTests.m:41: error: -[MLNMapViewScaleBarTests testShowsScale] : ((scaleBar.alpha) equal to (0)) failed: ("1") is not equal to ("0")
2023-07-02 14:36:16.056 xctest[66761:1090873] *** Assertion failure in -[MLNScaleBar preferredRow], MLNScaleBar.mm:294
<unknown>:0: error: -[MLNMapViewScaleBarTests testShowsScale] :  (NSInternalInconsistencyException)
Test Case '-[MLNMapViewScaleBarTests testShowsScale]' failed (0.008 seconds).
Test Suite 'MLNMapViewScaleBarTests' failed at 2023-07-02 14:36:16.059.
	 Executed 2 tests, with 3 failures (2 unexpected) in 0.016 (0.016) seconds
Test Suite 'MLNNSOrthographyAdditionsTests' started at 2023-07-02 14:36:16.059
Test Case '-[MLNNSOrthographyAdditionsTests testStreetsLanguages]' started.
Test Case '-[MLNNSOrthographyAdditionsTests testStreetsLanguages]' passed (0.000 seconds).
Test Suite 'MLNNSOrthographyAdditionsTests' passed at 2023-07-02 14:36:16.060.
	 Executed 1 test, with 0 failures (0 unexpected) in 0.000 (0.000) seconds
Test Suite 'MLNNetworkConfigurationTests' started at 2023-07-02 14:36:16.060
Test Case '-[MLNNetworkConfigurationTests testAccessingEventsFromMultipleThreads]' started.
Test Case '-[MLNNetworkConfigurationTests testAccessingEventsFromMultipleThreads]' passed (0.001 seconds).
Test Suite 'MLNNetworkConfigurationTests' passed at 2023-07-02 14:36:16.060.
	 Executed 1 test, with 0 failures (0 unexpected) in 0.001 (0.001) seconds
Test Suite 'ios_test.xctest' failed at 2023-07-02 14:36:16.060.
	 Executed 33 tests, with 27 failures (16 unexpected) in 100.821 (100.824) seconds
Test Suite 'All tests' failed at 2023-07-02 14:36:16.061.
	 Executed 33 tests, with 27 failures (16 unexpected) in 100.821 (100.826) seconds
warning: '-[MLNAnnotationViewTests testAnnotationViewInitWithFramePENDING]' is a pending test - skipping
warning: '-[MLNAnnotationViewTests testSelectingADisabledAnnotationViewPENDING]' is a pending test - skipping
2023-07-02 14:36:18,574 Deleting simulator 9C9E5E8C-69C9-401D-80AA-29E73411471B asynchronously.
2023-07-02 14:36:18,578 Done.
```

File: platform/ios/platform/darwin/test/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+objc_library(
+    name = "shared_tests_objcpp_srcs",
+    srcs = glob(
+        [
+            "*.h",
+            "*.mm",
+        ],
+        allow_empty = False,
+        exclude = [
+            "MLNOfflineStorageTests.mm",
```
Comment (louwers) [inline platform/ios/platform/darwin/test/BUILD.bazel:55]: This test was also disabled in the Xcode config.

File: platform/ios/platform/ios/test/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+objc_library(
+    name = "ios_tests_objc_srcs",
+    srcs = glob(
+        [
+            "*.h",
+            "*.m",
+        ],
+        allow_empty = False,
+        exclude = [
+            "MLNMapViewContentInsetTests.m",
```
Comment (louwers) [inline platform/ios/platform/ios/test/BUILD.bazel:17]: This test was also disabled in the Xcode config.

**Comment by louwers** [comment]
Setting the runner to the ordered runner fixes the problem with not being able to run the iOS tests through `bazel test`.

File: platform/ios/platform/darwin/test/MLNVersionNumber.m [hunk 3]
```diff
@@ -9,7 +9,7 @@ @interface MLNVersionTests : XCTestCase
 @implementation MLNVersionTests
 
 - (void)testVersionNumber {
-    XCTAssertGreaterThanOrEqual(MapboxVersionNumber, 0);
+    // XCTAssertGreaterThanOrEqual(MapboxVersionNumber, 0);
```
[the remark this answers is not in the document]
Comment (mwilsnd) [reply platform/ios/platform/darwin/test/MLNVersionNumber.m:12]: Weird, there is also a `MapboxVersionString`, both are exported by `Mapbox.h` in the macos/ios platform headers but never get initialized anywhere. Looks like dead code.

File: platform/ios/BUILD.bazel [hunk 4]
```diff
@@ -158,6 +158,7 @@ objc_library(
     name = "sdk",
     hdrs = _PUBLIC_HEADERS,
     data = ["resources"],
+    module_name = "Mapbox",
```
Comment (mwilsnd) [inline platform/ios/BUILD.bazel:161]: Do we want to rename this?

File: platform/ios/platform/darwin/test/MLNRendererConfigurationTests.m [hunk 5]
```diff
@@ -135,13 +135,7 @@ - (void)testSettingMLNIdeographicFontFamilyNameWithPlistValue {
     // `MLNIdeographicFontFamilyName` set to an invalid font family names array value
     {
         NSString *localFontFamilyName = [config localFontFamilyNameWithInfoDictionaryObject:@[@"test font 1", @"test font 2", @"test font 3"]];
-        
-        NSString *systemFontFamilyName;
-#if TARGET_OS_IPHONE
-        systemFontFamilyName = [UIFont systemFontOfSize:0 weight:UIFontWeightRegular].familyName;
-#else
-        systemFontFamilyName = [NSFont systemFontOfSize:0 weight:NSFontWeightRegular].familyName;
-#endif
```
Comment (mwilsnd) [inline platform/ios/platform/darwin/test/MLNRendererConfigurationTests.m:144]: If we want to fuse the platforms back together again, would we want to add this back for mac? Or just test with the local font?

Comment (louwers) [reply platform/ios/platform/darwin/test/MLNRendererConfigurationTests.m:144]: `systemFontFamilyName` does not get used in this block.

File: platform/ios/BUILD.bazel (hunk 4 above)
Comment (louwers) [reply platform/ios/BUILD.bazel:161]: I can rename this.

It affects how you import it in Swift code.

File: platform/ios/platform/darwin/test/MLNVersionNumber.m (hunk 3 above)
[the remark this answers is not in the document]
Comment (louwers) [reply platform/ios/platform/darwin/test/MLNVersionNumber.m:12]: Should I just delete this file maybe?

[the remark this answers is not in the document]
Comment (mwilsnd) [reply platform/ios/platform/darwin/test/MLNVersionNumber.m:12]: Yeah I guess I'd delete it. I'm not sure where the variables came from and a quick look at the blame history suggests it appeared during a platform re-org, but no sign of the variable initialization.

[the remark this answers is not in the document]
Comment (louwers) [reply platform/ios/platform/darwin/test/MLNVersionNumber.m]: Done.

**Review by mwilsnd — APPROVED** [review]

**Pull request merged**